src/modules/infobox/components/ImportContent.vue

```diff
@@ -0,0 +1,254 @@
+<template>
+    <div data-infobox="height-reference" class="import-overlay-content">
+        <div class="input-group d-flex">
+            <input
+                type="text"
+                class="form-control search-bar-input"
+                :placeholder="$t('import_online_placeholder')"
+                :value="importValue"
+                data-cy="import"
+                @input="onInputChange"
+                @focus="showProviders"
+                @keydown.down.prevent="goToFirstResult"
+                @keydown.esc.prevent="hideProviders"
+            />
+            <div v-if="listShown" ref="providers" class="providers-list-container">
+                <div class="providers-list">
+                    <div
+                        v-for="(provider, key) in filteredList"
+                        :key="provider"
+                        :tabindex="key"
+                        class="providers-list-item"
+                        @keydown.up.prevent="() => goToPrevious(key)"
+                        @keydown.down.prevent="() => goToNext(key)"
+                        @keydown.home.prevent="goToFirst"
+                        @keydown.end.prevent="goToLast"
+                    >
+                        <span
+                            v-html="provider.toShow"
+                            @click="() => onChooseProvider(provider.value)"
+                        ></span>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div
+            class="connect-btn-container"
+            :class="{
+                disabled: isConnectDisabled,
+            }"
+        >
+            <button
+                type="button"
+                class="btn btn-outline-secondary connect-btn"
+                :disabled="isConnectDisabled"
+                @click="onConnect"
+            >
+                {{ buttonText }}
+            </button>
+        </div>
+        <!-- todo just for test -->
+        <div class="d-flex flex-column h-50 overflow-y-auto">
+            <div
+                v-for="layer in importedLayers"
+                :key="layer.externalLayerId"
+                @click="() => addLayer(layer)"
+                class="cursor-pointer"
+            >
+                {{ layer.name }}
+            </div>
+        </div>
```

Comment:
Do you plan to finish this in a later PR ? It seems to work but there still some styling work to do.

Reply:
Ok for this, @pakb told me that this part will be done in a later ticket/pr